deployer.sh

```diff
@@ -272,14 +270,86 @@ if [[ $RUN_TESTS != 0 ]] ; then
     qxCmd=test
 fi
 for repo in $ENABLED_REPOS ; do
-    [[ $repo == "qooxdoo" || $repo == "qooxdoo-compiler" ]] && continue
+    [[ $repo == "qooxdoo" ]] && continue
+
+    info "compile and test $repo"
     repoDir=${REPO_DIRS[$repo]}
     pushDirSafe $repoDir
     $WORKING_ABS_DIR/bin/qx $qxCmd $QX_COMPILE_ARGS
     popDir
 done
 
+info "Bootstrap is resolved and repos are compiled"
 
-echo "Bootstrap is resolved and repos are compiled"
+# fill .npmrc with access token
+echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN:-}" > ~/.npmrc
+PACKAGE_DATE=$(date +%Y%m%d-%H%M)
+[[ ! -d $WORKING_ABS_DIR/deploy ]] && mkdir -p $WORKING_ABS_DIR/deploy
+rm -fR $WORKING_ABS_DIR/deploy/*
 
+function publishFramework {
+    info "publish qooxdoo framework"
+
+    pushDirSafe ${REPO_ABS_DIRS["qooxdoo"]}
+    local VERSION=$(cat version.txt)
+    VERSION="$VERSION-$PACKAGE_DATE"
```

Comment:
When v6 is released, the package date suffix needs to go. We have time until then, but maybe you want to make it conditional already now, maybe dependent on whether "alpha" or "beta" is part of the version? Then the suffix would be added automatically as soon as a alpha or beta version is being published.

Reply:
How can be this done in bash?

Reply:
```bash
if [[ "$VERSION" =~ (alpha|beta) ]]; then
  VERSION="$VERSION-$PACKAGE_DATE"
fi
```

Reply:
The question is if we should rely on `version.txt` at all as the authoritative version info. But we can solve that later...